conf.yaml

```diff
@@ -996,10 +996,10 @@ contents:
                 path:   docs/asciidoc
           - title:      Elastic Common Schema (ECS) Reference
             prefix:     en/ecs
-            # Please do not update current to 8.12
-            current:    8.11
-            branches:   [ {main: master}, 8.11, 8.10, 8.9, 8.8, 8.7, 8.6, 8.5, 8.4, 8.3, 8.2, 8.1, 8.0, 1.12, 1.11, 1.10, 1.9, 1.8, 1.7, 1.6, 1.5, 1.4, 1.3, 1.2, 1.1, 1.0 ]
-            live:       [ 8.11, 1.12 ]
+            # Please do not update current to 8.17
+            current:    8.16
```

Comment:
I don't think we're supposed to update this yet? Only once 8.16 goes live.

Reply:
I've reverted that for now